src/constitutive_relations/eos/CMakeLists.txt

```diff
@@ -9,17 +9,19 @@ include_directories(${ATS_SOURCE_DIR}/src/factory)
 
 add_library(relations_eos
   eos_evaluator.cc
+  eos_evaluator_tp.cc
+  eos_evaluator_ctp.cc    
   isobaric_eos_evaluator.cc
   eos_factory.cc
-  eos_constant.cc eos_linear.cc eos_ideal_gas.cc eos_water.cc eos_ice.cc eos_vapor_in_gas.cc
+  eos_constant.cc eos_linear.cc eos_ideal_gas.cc eos_water.cc eos_ice.cc eos_vapor_in_gas.cc eos_sw.cc
   viscosity_evaluator.cc
   viscosity_relation_factory.cc
   viscosity_constant.cc
   viscosity_water.cc
   molar_fraction_gas_evaluator.cc
   vapor_pressure_relation_factory.cc
   vapor_pressure_water.cc
-  effective_pressure_evaluator.cc
+  effective_pressure_evaluator.cc effective_concentration_evaluator.cc
```

Comment:
I think the effective_pressure_evaluator can be removed.  That was only to deal with evaluators that expected that liquid pressure >= p_atm (for instance compressibility and equations of state), but I believe that all of those evaluators were updated to have max(p, p_atm) in them directly.

Reply:
Removed